src/App.jsx

```diff
@@ -14,33 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const SORT_FIELD_BY_ALPHABET = 'alphabet';
+const SORT_FIELD_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+function sortedGoods(goods, { sortField }) {
```

Comment:
```suggestion
function getSortedGoods(goods, { sortField }) {
```
sortedGoods sounds like it's already an array of sorted goods. Functions should contain a verb in the name - this function returns an array so `get` would be the correct word here